src/components/SubmitButtonStyled.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const Button = styled.input`
```

Comment:
could you also please put the button on centre of the page as the document shows it on centre

Reply:
This is styling related - we are going to take care of it later.